admin_tests/nodes/test_views.py

```diff
@@ -341,3 +343,70 @@ def test_confirm_registration_as_ham(self):
 
         self.registration.refresh_from_db()
         nt.assert_true(self.registration.spam_status == 4)
+
+
+class TestAdminNodeLogView(AdminTestCase):
+
+    def setUp(self):
+        super(TestAdminNodeLogView, self).setUp()
+
+        self.request = RequestFactory().post('/fake_path')
+        self.user = AuthUserFactory()
+        self.auth = Auth(self.user)
+        self.node = ProjectFactory(creator=self.user)
+        self.node.save()
+
+    def test_get_object(self):
+
+        view = AdminNodeLogView()
+        view = setup_log_view(view, self.request, guid=self.node._id)
+
+        nt.assert_true(self.node, view.get_object())
+
+    def test_get_queryset(self):
+
+        self.node.set_title('New Title', auth=self.auth, save=True)
+
+        view = AdminNodeLogView()
+        view = setup_log_view(view, self.request, guid=self.node._id)
+
+        logs = view.get_queryset()
+
+        log_entry = logs.all()[0]
+        nt.assert_true(log_entry.action == 'edit_title')
+        nt.assert_true(log_entry.params['title_new'] == u'New Title')
+
+    def test_get_context_data(self):
+
+        self.node.set_title('New Title', auth=self.auth, save=True)
+
+        view = AdminNodeLogView()
+        view = setup_log_view(view, self.request, guid=self.node._id)
+
+        logs = view.get_context_data()['logs']
+        log_entry = logs[0][0]
+        log_params = logs[0][1]
+        nt.assert_true(log_entry.action == NodeLog.EDITED_TITLE)
+        nt.assert_true((u'title_new', u'New Title') in log_params)
+        nt.assert_true((u'node', self.node._id) in log_params)
+
+    def test_get_logs_for_children(self):
+        """ The "create component" action is actually logged as a create_project action
+        for it's child with a log parameter 'node' having it's guid as a value. We have to ensure
+        that all the components a parent has created appear in it's admin app logs, so we can't just
```

Comment:
very minor non-blocking grammar: 
it's = it is
its = possessive

the three it's above should be its